bedrock/teach/templates/teach/smarton/security.html

```diff
@@ -0,0 +1,325 @@
+            <p>{{ _('These add-ons will help you identify what you’re clicking before you click:') }}</p>
+            <ul>
+              <li>
+                {{ _('<a href="{url}">Long URL please</a> — replaces most shortened URLs with the originals so you know the destination').format(url='https://addons.mozilla.org/firefox/addon/long-url-please/')|safe }}
+              </li>
+              <li>
+                {{ _('<a href="{url}">URL Tooltip</a> — displays the destination of a link when hovering over it').format(url='https://addons.mozilla.org/firefox/addon/url-tooltip/')|safe }}
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+    </div>
+  </section>
+
+  <section id="chat" class="section section-chat">
+    <div class="content">
+
+      <h2 class="section-title">{{ _('Blast your cybersecurity smarts') }}</h2>
+      <p class="section-intro">
+        {{ _('The more we talk, share tips and stay alert about cybersecurity, the safer we all are.') }}
+        {{ _('Keep the conversation going with your community on social media, at home — maybe even on that public Wi-Fi at the airport.') }}
+      </p>
+
+      <div class="with-sidebar">
+        <aside class="callout">
+          <section>
+            <p>{{ _('Connect with your community to explore topics related to privacy and tracking by joining or starting a <a href="{url}">Mozilla Club</a>.').format(url='https://teach.mozilla.org/clubs/')|safe }}</p>
+          </section>
+
+          <section>
+            <p>{{ _('Are you a teacher? Give your students a <a href="{url}">simple lesson in cybersecurity</a>.').format(url='https://teach.mozilla.org/activities/')|safe }}</p>
+          </section>
+          {% include 'teach/smarton/includes/scribble-sm.svg' %}
+        </aside>
+
+        <ol class="big-list">
+          <li>
+            <p>
+              {{ _('How many online passwords do you have?') }}
+              <a class="tweet-this" href="{{ 'https://www.twitter.com/intent/tweet?url=%s&amp;text=%s'|format('https://mzl.la/foo'|urlencode, _('How many online passwords do you have?')|urlencode)|e }}">{{ _('Tweet this') }}</a>
+            </p>
+            {% include 'teach/smarton/includes/list-number-1.svg' %}
+          </li>
+          <li>
+            <p>
+              {{ _('What kind of security tips and tricks do you use online?') }}
+              <a class="tweet-this" href="{{ 'https://www.twitter.com/intent/tweet?url=%s&amp;text=%s'|format('https://mzl.la/foo'|urlencode, _('What kind of security tips and tricks do you use online?')|urlencode)|e }}">{{ _('Tweet this') }}</a>
+            </p>
+            {% include 'teach/smarton/includes/list-number-2.svg' %}
+          </li>
+          <li>
+            <p>
+              {{ _('On a scale from 1 to 10, how safe do you feel online?') }}
+              {{ _('Is it always clear to you whether or not your activities are secure?') }}
```

Comment:
Sorry, missed this one: it's a tweet, let's keep it in one string for sake of clarity and shortness.
```
{{ _('On a scale from 1 to 10, how safe do you feel online? Is it always clear to you whether or not your activities are secure?') }}
```